Makefile

```diff
@@ -1,38 +1,7 @@
-PREFIX=/usr/local
```

Comment:
Why `make nsqd build` we don't use?
Also I suggest do not touch/remove the rest logic